cmd/minikube/cmd/start_flags.go

```diff
@@ -342,6 +344,7 @@ func generateClusterConfig(cmd *cobra.Command, existing *config.ClusterConfig, k
 			HypervUseExternalSwitch: viper.GetBool(hypervUseExternalSwitch),
 			HypervExternalAdapter:   viper.GetString(hypervExternalAdapter),
 			KVMNetwork:              viper.GetString(kvmNetwork),
+			KVMPrivateNetwork:       viper.GetString(kvmPrivateNetwork),
```

Comment:
don't we need to add a clean up logic ? so when we delete minikube we delete the created networks too ?

Reply:
that should already be an automated part of the delete - pls have a look at the cleanup (last) section in the example given in the issue's description above

Reply:
this is the code segment that takes care of that:
https://github.com/kubernetes/minikube/blob/ff0e25ada888cf1bd7b262857ce96531c1231e69/pkg/drivers/kvm/kvm.go#L427-L442